sdk/core/Azure.Core.TestFramework/src/Instrumentation/InstrumentResultInterceptor.cs

```diff
@@ -41,7 +41,7 @@ public void Intercept(IInvocation invocation)
 
             if (
                 // Generated ARM clients will have a property containing the sub-client that ends with Operations.
-                (invocation.Method.Name.StartsWith("get_") && (type.Name.EndsWith("Operations") || type.BaseType.Name.EndsWith("Operations"))) ||
+                (invocation.Method.Name.StartsWith("get_") && (type.Name.EndsWith("Operations") || (type.BaseType != null && type.BaseType.Name.EndsWith("Operations")))) ||
```

Comment:
Why did this start failing in existing data-plane tests?

Reply:
that code there doesn't exist in master yet.  If you remember I took our interceptor and merged with with data plane to share.  Before the merge we never hit this code without a basetype, but now that its shared we do, so needed to add the additional check

Reply:
Gotcha!